src/graphql.tsx

```diff
@@ -635,7 +635,10 @@ class ObservableQueryRecycler {
   public recycle (observableQuery: ObservableQuery<any>): void {
     // Stop the query from polling when we recycle. Polling may resume when we
     // reuse it and call `setOptions`.
-    observableQuery.stopPolling();
+    observableQuery.setOptions({
+      fetchPolicy: 'cache-only',
```

Comment:
Sure!
I thought that `setOptions` where strictly overriding the old options but I just noticed that it doesn't. As far as I can tell there's no explicit default for `fetchPolicy` and the current `setOptions` implementation doesn't allow removing previously set properties:

```typescript
    const oldOptions = this.options;
    this.options = {
      ...this.options,
      ...opts,
    } as WatchQueryOptions;
```

Ideally I'd like to remove both the `pollInterval` and `fetchPolicy` completely before applying `options` inside `reuse`.

Any ideas? I'll commit the new breaking test case for now (which is testing for strict equality of options before and after recycle).